### ISSUE TITLE
Wrong abrev_state value from lookup_muni

### ISSUE THREAD

**Issue by allan4229** [ipeaGIT/geobr#134]
The following code returns 'PE' instead of 'PR' for *abrev_state* column :
```
> lookup_muni(name="Curitiba")
  |=============================================================================================================================================================================================================================================| 100%
Returning results for municipality Curitiba
     code_muni name_muni code_state name_state abrev_state code_micro name_micro code_meso                 name_meso code_immediate name_immediate code_intermediate name_intermediate
4005   4106902  Curitiba         41     Paraná          PE      41037   Curitiba      4110 Metropolitana de Curitiba         410001       Curitiba              4101          Curitiba
```

Expected result :
```
... abrev_state ...
... PR ...
```

Versions :

|name|version|
|----|-------|
|R|3.6.3|
|geobr (test1)|default (install.packages)|
|geobr (test2)|git (1.2-1 1db08f3)|

Seems like the same problem described in #87 .

**Comment by rafapereirabr** [comment]
Thanks for the heads up! @kauebraga , could please have a look into this?

**Comment by kauebraga** [comment]
Fixed at aca40704170b5d4dc5ac9e3fd81228ad4e7b1ed4

**Comment by rafapereirabr** [comment]
Hi @allan4229 . @kauebraga  and I have updated the file and this issue has been fixed. Thank you again for pointing that out.
